src/URLHelper.php

```diff
@@ -427,23 +428,20 @@ private static function is_internal_content($url)
     }
 
     /**
-     * Determines if URL is an external URL.
+     * Checks whether an URL is external.
      *
-     * True if `$url` is an external url or subdomain (http://cdn.example.org = true). False if
-     * relative or local true if it's a subdomain
+     * True if the URL is an external URL or subdomain (http://cdn.example.org = true). False if
+     * the URL is relative or local, true if it’s a subdomain.
```

Comment:
We might want to enhance the docblock a bit here. We are listing subdomain twice here which is a bit confusing @mcaskill , are you able to clear that up as well?